tests/framework/InternalParallelTests/test_internal_parallel_PP_LS.xml

```diff
@@ -130,14 +126,14 @@
       <Model class="Models" type="PostProcessor">LimitSurfaceIntegralWeighted</Model>
       <Output class="DataObjects" type="PointSet">LimitSurfaceWeightedPb</Output>
       <Output class="OutStreams" type="Print">LimitSurfaceWeightedPb_dump</Output>
-      <Output class="Files" type="">LimitSurfaceWeightedPb_FileObj.csv</Output>
+      <!-- NOT ALLOWED ANYMORE: <Output class="Files" type="">LimitSurfaceWeightedPb_FileObj.csv</Output> -->
```

Comment:
Are these comments meant to be left for some particular length of time as an example to previous users?

Reply:
Let's keep it for now.